### DISCUSSION TITLE
Table: How to customize emptyContent

### DISCUSSION THREAD

**Discussion by manuel-mauky** [heroui-inc/heroui#3863] [Q&A] [answered]
The Table allows you to define `emptyContent` on the `<TableBody>` component to show when no elements are available.
However, the component you pass in there is rendered in a `<td>` element which has various styles assigned that you cannot customize. Especially the `h-40` class is an issue in my case. 
![image](https://github.com/user-attachments/assets/042cab5b-4482-42f6-a23c-2d9b8c41331d)

I'd like to have a smaller height of the empty table but cannot find a way to achieve this.

Current result:
![image](https://github.com/user-attachments/assets/620df60a-1bd0-4094-bdf1-9325e35550ab)

Goal:
<img width="479" alt="image" src="https://github.com/user-attachments/assets/c368cbd3-402b-4dda-be20-a544227140c5">

Is it somehow possible to customize the classes on that `<td>` element? I've tried to put classes on the `<TableBody>` but without success.

**Comment by AnYiEE** [comment, marked as answer]
```tsx
<Table
	classNames={{
		emptyWrapper: 'h-auto',
	}}
>
	<TableBody emptyContent={}></TableBody>
</Table>
```

**Reply by manuel-mauky** [reply]
Thanks a lot. This solves it. I haven't had the "slots" thing on my radar but now I think I understand them.
